colcon_core/task/python/test/pytest.py

```diff
@@ -40,6 +40,10 @@ def add_arguments(self, *, parser):  # noqa: D102
             help='Pass arguments to pytests. '
             'Arguments matching other options must be prefixed by a space,\n'
             'e.g. --pytest-args " --help"')
+        parser.add_argument(
+            '--pytest-with-coverage',
+            action='store_true',
+            help='Generate coverage information')
```

Comment:
@dirk-thomas nit^2: 
```suggestion
            help='Generate pytest coverage information')
```

Reply:
The generated coverage data isn't specific to `pytest`. Therefore I didn't include it in the help text.

Reply:
That's a fair point, disregard.